modules/expo-location/src/Location.js

```diff
@@ -68,7 +68,12 @@ function getProviderStatusAsync(): Promise<ProviderStatus> {
   return Location.getProviderStatusAsync();
 }
 
-async function getCurrentPositionAsync(options: LocationOptions = {}): Promise<LocationData> {
+async function getCurrentPositionAsync(options: {
+  timeout?: number,
+  enableHighAccuracy?: boolean,
+  maximumAge?: number,
```

Comment:
why not as `LocationOptions` type?

Reply:
I got no idea what is causing https://github.com/expo/expo/issues/2226#issuecomment-427272746 . More specific info is needed here as I've never encountered such problem on testing devices.

Reply:
I've changed this, because I've added this `timeout` option, but I'll reverse it